imports/api/PointCalculationUtils.js

```diff
@@ -0,0 +1,106 @@
+import { Constants } from './Constants';
+import { NewGameUtils } from './NewGameUtils';
+
+export var PointCalculationUtils = {
+	jpn: {
+		dealin_delta,
```

Comment:
Perhaps these should be renamed jpn_dealin_delta to more easily accomodate later addition of hkg